src/source/remote.js

```diff
@@ -156,7 +156,7 @@ function maybeWrapInBlockedSource(source, { blockSize, cacheSize }) {
   if (blockSize === null) {
     return source;
   }
-  return new BlockedSource(source, blockSize, cacheSize);
+  return new BlockedSource(source, { blockSize, cacheSize });
```

Comment:
Oh wow, kind of a crucial fix to geotiff regardless of changes within `BlockedSource`...

Reply:
I had forgotten about this! Also, I had forgotten about https://github.com/geotiffjs/geotiff.js/pull/224#issuecomment-897772973 so I'll need to resolve that.  Sorry for putting it in the wrong PR

Reply:
It appears [the issue about logging](https://github.com/geotiffjs/geotiff.js/pull/224#issuecomment-897772973) has been noticed by others where the [error just always gets logged out](https://github.com/geotiffjs/geotiff.js/issues/218#issuecomment-862281272)...it must be somewhere in geotiff since even if I try-catch it as mentioned, it does nothing.